corehq/apps/hqwebapp/views.py

```diff
@@ -540,7 +547,10 @@ def login_new_window(request):
 @location_safe
 @login_required
 def iframe_domain_login_new_window(request):
```

Comment:
it is still an iframe for non-sso users.

Reply:
I think a function should be named after what it does 100% of the time, otherwise it starts violating the principle of least surprise. If I'm unfamiliar with this code and see `iframe_domain_login_new_window`, aren't I going to be expecting to see an iframe?

Reply:
ok i addressed this and renamed to `domain_login_new_window`. Is there anything else blocking approval?